test/channel_permissions.js

```diff
@@ -18,13 +28,13 @@ class Context {
 	}
 
 	async setup() {
-		await this.authClient.updateUser(this.adminUser);
-		await this.authClient.updateUser(this.moderator);
-		await this.authClient.updateUser(this.guestUser);
-		await this.authClient.updateUser(this.channelMember);
-		await this.authClient.updateUser(this.channelOwner);
-		await this.authClient.updateUser(this.messageOwner);
-		await this.authClient.updateUser(this.scapegoatUser);
+		await this.authClient.upsertUser(this.adminUser);
```

Comment:
That's good but maybe do it separately since there might be more cases of it. Otherwise, reason isn't clear.

Reply:
might note it somewhere